docs/getting-started.md

```diff
@@ -17,11 +17,25 @@ docker-compose up -f docker/docker-compose.yml -d
 
 ### Account creation
 
-TBD
+Use the Mainflux manager HTTP/REST API to create user account:
+
+```
+curl -s -S -i -X POST -H "Content-Type: application/json; charset=utf-8" localhost:8180/users -d '{"email":"john.doe@email.com", "password":"123"}'
+```
```

Comment:
I would also add `HTTPS` variant served by NginX (i.e. port `443` or `80`->`443` redirection)

Reply:
It does not have to be for every example, but at least one - to show people how to use HTTPS + cert